build.gradle

```diff
@@ -97,6 +97,7 @@ subprojects {
         testLogging() {
             events 'failed'
             exceptionFormat 'full'
+            // showStandardStreams = true
```

Comment:
?

Reply:
I keep forgetting how to enable stdout in the console. Is that ok if I leave it commented?